src/backend/defs/rest-api/objects/MessageV2.yaml

```diff
@@ -0,0 +1,59 @@
+---
+type: object
+properties:
```

Comment:
I don't get where is defined the author of the message, it can be:
* the user (in a specific LocalIdentity)
* a known contact 
* an unknown address

(I understood that a message can still have multiple recipients others than me, which can be known or not)